bin/geni-sync-wireless

```diff
@@ -0,0 +1,618 @@
+                'lead_id' : row.lead_id,
+                'project_name' : row.project_name,
+                'project_description' : row.project_purpose,
+                'members' : []
+                }
+
+        # Get all members of WIMAX-enabled projects
+        query = self._session.query(self.PROJECT_MEMBER_TABLE.c.member_id,
+                                    self.PROJECT_MEMBER_TABLE.c.project_id)
+        query = query.filter(self.PROJECT_MEMBER_TABLE.c.project_id.in_(\
+                project_ids))
+        if self._user:
+            query = query.filter(self.PROJECT_MEMBER_TABLE.c.member_id == \
+                                     self.MEMBER_ATTRIBUTE_TABLE.c.member_id)
+            query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.name == \
+                                     'username')
+            query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.value == \
+                                     self._user)
+        member_rows = query.all()
+        for row in member_rows:
+            projects[row.project_id]['members'].append(row.member_id)
+
+        # Don't return any projects with no members 
+        # (if we're filtering by user)
+        for project_id, project_info in projects.items():
+            if len(project_info['members']) == 0:
+                del projects[project_id]
+
+        self._geni_projects = projects
+
+    # Grab info about all people in wimax projects
+    def get_geni_members(self):
+
+        projects = self._geni_projects
+
+        members = {}
+
+        # Get unique list of all member_ids over all projects
+        member_ids =  set()
+        for proj_id, project_info in projects.items():
+            for member_id in project_info['members']:
+                member_ids.add(member_id)
+
+        # add the holdingpen admin, who may not be the member of any project
+        query = self._session.query(self.MEMBER_ATTRIBUTE_TABLE.c.member_id)
+        query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.name == 'username')
+        query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.value == \
+                                 self._options.holdingpen_admin)
+        holdingpen_admin_rows = query.all()
+        for row in holdingpen_admin_rows:
+            member_ids.add(row.member_id)
+
+        # Add the project leads
+        for proj_id, project_info in projects.items():
+            lead_id = project_info['lead_id']
+            member_ids.add(lead_id)
+
+        # Turn set back into list, to grab all users with these member ID's
+        member_ids = list(member_ids)
+
+        query = self._session.query(self.MEMBER_ATTRIBUTE_TABLE)
+        query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.member_id.in_(\
+                member_ids))
+        query = query.filter(self.MEMBER_ATTRIBUTE_TABLE.c.name.in_(\
+                ["username", 'member_enabled', "first_name", "email_address", 
+                 "last_name", "displayName"]))
+        
```

Comment:
Why not right here or in ensure_project_members_exist avoid creating orbit entries for disabled users?